roles/proxy/tasks/RedHat.yml

```diff
@@ -14,5 +14,5 @@
     path: /etc/yum.conf
     section: main
     option: proxy
-    value: "{{ proxy_proxies['https']default(proxy_proxies['http']) }}"
+    value: "{{ proxy_proxies['https']|default(proxy_proxies['http']) }}"
```

Comment:
Please add spaces around the `|`, I think that's what the linter's `var-spacing` check would say, too.

Reply:
The linter meant that there was no Pipe before. But yes i agree that it is better to add the spaces too. :)